### ISSUE TITLE
Default job type settings

### ISSUE THREAD

**Issue by dfaller** [ngageoint/scale#665]
Add the capability to define default settings valued for job types.

1. Create a new job type model JSON field called "default_settings". We'll want to consider creating a new JSON schema for this.
2. At schedule time, pull the settings from the job type and populate the settings in the job execution job_configuration.

**Comment by dfaller** [comment]
For the "default_settings" field, just have a simple JSON schema:

{
  "setting_name_1": "setting_value_1",
  "setting_name_2": "setting_value_2"
}

Names and values must be strings.

**Comment by dfaller** [comment]
New plan:

Make the new job type field "configuration" with a schema that looks like this:
```
{
  "version": "1.0",
  "default_settings": {
    "setting_name_1": "setting_value_1",
    "setting_name_2": "setting_value_2"
  }
}
```

**Comment by mikenholt** [comment]
fixed with PR #672
